src/ol/interaction/drawinteraction.js

```diff
@@ -715,6 +715,29 @@ ol.interaction.Draw.prototype.abortDrawing_ = function() {
 
 
 /**
+ * Continue drawing a feature.  This only works on line features, that is
+ * features with a `LINE_STRING` geometry.
+ * @param {!ol.Feature} feature Feature to continue drawing.
```

Comment:
Is the `!` really needed? It's the only function in this class to have this no-null annotation